### PR TITLE
Upgrade twilio libs

### PR DESCRIPTION
**Pull request by juancarlospaco**
- Upgrade twilio libs.
- Fix test.

### CODE REVIEW

File: public/js/tests/tests.js [hunk 1]
```diff
@@ -1,90 +1,74 @@
...
-
-  mock.verify();
-  ok(true);
+const indexPageAsHtml = window.__html__['tests.html'].trim();
+document.body.innerHTML = indexPageAsHtml;
+
+
+describe('TwilioChat', function() {
+  context('#sortChannelsByName()', function() {
+    it('Sorts channels by name', function() {
```
Comment (acamino) [inline public/js/tests/tests.js]: Why did you change the casing for the description? We spent time trying to get a solid version of this file in the Node.js version of this tutorial, right? There is no benefit on introducing this kind of changes.

File: public/js/tests/tests.js [hunk 2]
```diff
@@ -1,90 +1,74 @@
...
+      const channels = [
```
Comment (acamino) [inline public/js/tests/tests.js]: We're not using a compiler like [babel](https://babeljs.io/) to use some of the ES6 syntax, unless I totally missed it. This kind of changes will break this implementation in a few browsers.

File: public/js/tests/tests.js (hunk 1 above)
Comment (juancarlospaco) [inline public/js/tests/tests.js]: Done.

File: public/js/tests/tests.js (hunk 2 above)
Comment (juancarlospaco) [inline public/js/tests/tests.js]: Done.
